### PR TITLE
New project from existing

### PR DESCRIPTION
**Pull request by mml2222**
It's not a completed branch. Was able to determine when to show the different buttons. Still needs to be connected properly between modals and write emotions into database.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -13,13 +13,14 @@
 
     <!-- MOUNT POINT -->
     <app></app>
+    <!-- <finalReflection></finalReflection> -->
```
Comment (lrbloch) [inline index.html]: remove comment ;)

File: tags/create.tag [hunk 2]
```diff
@@ -64,23 +73,50 @@
...
+        if (querySnapshot.empty === true) {
+          this.firstProject = true;
+          observer.trigger('project:firstProject', this.firstProject);
+        }
+        else {
+          this.newProject = true;
+          observer.trigger('project:newProject', this.newProject);
+        }
+      });
+    }
```
Comment (lrbloch) [inline tags/create.tag]: if we have this function (`checkProject`), why do we need the observers to set firstProject?

File: tags/login.tag [hunk 3]
```diff
@@ -31,43 +22,52 @@
...
-  // console.log(a);
-
+        this.user = null;
+        stopListening();
+      }
+      this.checkProject();
+      this.update();
+    });
+    // check if have Projects
+    checkProject(){
```
Comment (lrbloch) [inline tags/login.tag:58]: this function exists in 2 different tags. is that on purpose?

File: tags/create.tag (hunk 2 above)
Comment (mml2222) [reply tags/create.tag]: Yes, I thought I need it in two places but it's not necessary.

File: tags/login.tag (hunk 3 above)
Comment (mml2222) [reply tags/login.tag:58]: The function should be in this tag not in the create tag.

**Review by lrbloch — APPROVED** [review]

**Pull request merged**